### PR TITLE
Data Hooks Import

### PR DESCRIPTION
**Pull request by jbkolze**
Import the following data hooks from groundwork and refactor to use cwmsjs:
- useCdaCatalog
- useCdaLatestValue
- useCdaLocation
- useCdaTimeSeries
- useNwpsGauge
- useNwpsGaugeData

### CODE REVIEW

**Comment by jbkolze** [comment]
@krowvin Can you take a glance a let me know if you're happy with how the docs pages are set up?  And any thoughts on the hooks themselves are also welcome.

File: lib/index.jsx [hunk 1]
```diff
@@ -1,9 +1,26 @@
 // Import components
-import TSPlot from './components/data/plots/TSPlot';
-import TSTable from './components/data/tables/TSTable';
-import GageMap from "./components/data/maps/GageMap"
+import TSPlot from "./components/data/plots/TSPlot";
```
Comment (krowvin) [inline lib/index.jsx:2]: Good catch, glad you're helping keep track of the standards!

File: package.json [hunk 2]
```diff
@@ -65,5 +65,8 @@
         "postcss": "^8.4.38",
         "tailwindcss": "^3.4.4",
         "vite": "^5.3.1"
+    },
+    "peerDependencies": {
+        "@tanstack/react-query": "^5.51.15"
```
Comment (krowvin) [inline package.json:70]: Should plotly also be in the peer dependency? 

https://github.com/USACE-WaterManagement/groundwork-water/blob/3e01ca9f4d5b70144a5dd54e492c69826f3dcdac/vite.config.js#L16

**Review by krowvin — APPROVED** [review]
I looked through all your changes. I have no major issues. 

The only thing i'll followup on is if I need to be putting plotly in the peer dependencies as well. Not just the external part of the vite config for rollup.

**Comment by krowvin** [comment]
I'll need to publish this to NPM before the install will complete. I'll wait to merge until I can bump the version and update NPM.

**Comment by jbkolze** [comment]
> The only thing i'll followup on is if I need to be putting plotly in the peer dependencies as well. Not just the external part of the vite config for rollup.

Forgot to mention that -- React and ReactDOM should definitely be moved into peerDependencies.  I think that generally applies for anything that's external, so probably plotly and OpenLayers as well if that's the route we're going with them.

**Pull request merged**